src/EnergyPlus/UnitarySystem.cc

```diff
@@ -10133,7 +10326,7 @@ namespace UnitarySystems {
                                         Psychrometrics::PsyHFnTdbW(RefTemp, MinHumRatio)) -
                          this->m_SenLoadLoss;
 
-            if (this->m_Humidistat) {
+            if (this->m_Humidistat || this->m_CoolingCoilSubType_Num == DataHVACGlobals::CoilDX_SubcoolReheat) {
```

Comment:
Why isn't this same logic used if an ATMixer exists at lines 10294-10322? Does the subcool coil always calculate or report LatOutput?

Reply:
@rraustad I am not clear for your question. Here is my concern of latent load calculation using hg to ensure that a sum of sensible and latent is equal to enthalpy differences, when the sensible is calculated using min W. I have a document to address this issue and was discussed in the sizing calls.

We can discuss this issue as a follow up.

Reply:
LatOutput is calculated always in subcool coil model.

Reply:
My question is why didn't you use this same logic if there is an ATMixer? Look at lines 10294-10322

 if(this->m_Humidistat || this->m_CoolingCoilSubType_Num == DataHVACGlobals::CoilDX_SubcoolReheat)

Reply:
@rraustad  Could you make a copy of lines 10294-10322?

Reply:
You could just go to line 10294 of UnitarySystem.

        if (this->ATMixerExists) {
            if (this->ATMixerType == DataHVACGlobals::ATMixer_SupplySide) {
                // Air terminal supply side mixer
                int ATMixOutNode = this->ATMixerOutNode;
                SensOutput =
                    DataLoopNode::Node(ATMixOutNode).MassFlowRate * (Psychrometrics::PsyHFnTdbW(DataLoopNode::Node(ATMixOutNode).Temp, MinHumRatio) -
                                                                     Psychrometrics::PsyHFnTdbW(RefTemp, MinHumRatio) - this->m_SenLoadLoss);
                if (this->m_Humidistat) {
                    //   Calculate latent load met (at constant temperature)
                    LatOutput = DataLoopNode::Node(ATMixOutNode).MassFlowRate *
                                    (Psychrometrics::PsyHFnTdbW(RefTemp, DataLoopNode::Node(ATMixOutNode).HumRat) -
                                     Psychrometrics::PsyHFnTdbW(RefTemp, RefHumRat)) -
                                this->m_LatLoadLoss;
                } else {
                    LatOutput = 0.0;
                }
            } else {
                // Air terminal inlet side mixer
                SensOutput = AirMassFlow * (Psychrometrics::PsyHFnTdbW(DataLoopNode::Node(OutletNode).Temp, MinHumRatio) -
                                            Psychrometrics::PsyHFnTdbW(RefTemp, MinHumRatio)) - this->m_SenLoadLoss;
                if (this->m_Humidistat) {
                    //   Calculate latent load met (at constant temperature)
                    LatOutput = AirMassFlow * (Psychrometrics::PsyHFnTdbW(RefTemp, DataLoopNode::Node(OutletNode).HumRat) -
                                               Psychrometrics::PsyHFnTdbW(RefTemp, RefHumRat)) -
                                this->m_LatLoadLoss;
                } else {
                    LatOutput = 0.0;
                }
            }
        } else {
            // Calculate sensible load met (at constant humidity ratio)
            SensOutput = AirMassFlow * (Psychrometrics::PsyHFnTdbW(DataLoopNode::Node(OutletNode).Temp, MinHumRatio) -
                                        Psychrometrics::PsyHFnTdbW(RefTemp, MinHumRatio)) -
                         this->m_SenLoadLoss;

            if (this->m_Humidistat || this->m_CoolingCoilSubType_Num == DataHVACGlobals::CoilDX_SubcoolReheat) {

                //   Calculate latent load met (at constant temperature)
                LatOutput = AirMassFlow * (Psychrometrics::PsyHFnTdbW(RefTemp, DataLoopNode::Node(OutletNode).HumRat) -
                                           Psychrometrics::PsyHFnTdbW(RefTemp, RefHumRat)) -
                            this->m_LatLoadLoss;
            } else {
                LatOutput = 0.0;
            }
        }

Reply:
@rraustad I understand your concern. In order to be consistent with ATMMixer, I set this->m_Humidistat = true, when this->m_CoolingCoilSubType_Num == DataHVACGlobals::CoilDX_SubcoolReheat.

Reply:
If you setm_Humidistat = true then the code will look for a zone with a humidistat.

                if (thisSys.m_Humidistat && thisSys.m_ControlType == ControlType::Load) {
                    bool AirNodeFound = false;
                    for (int HStatZoneNum = 1; HStatZoneNum <= DataZoneControls::NumHumidityControlZones; ++HStatZoneNum) {
                        if (DataZoneControls::HumidityControlZone(HStatZoneNum).ActualZoneNum != thisSys.ControlZoneNum) continue;
                        AirNodeFound = true;
                    }
                    if (!AirNodeFound && thisSys.ControlZoneNum > 0) {
                        ShowSevereError("Input errors for " + cCurrentModuleObject + ":" + thisObjectName);

Reply:
@rraustad I agree. I did a temporary fix. LatOutput should be processed as the same sa SenOutput.